bin/vue

```diff
@@ -14,6 +14,7 @@ program
 program
   .arguments('<command>')
   .action((cmd) => {
+    if (['init', 'list', 'build'].includes(cmd)) return
```

Comment:
This just removes the redundant wanring message, and is not enough to solve `Failed to download repo vuejs-templates/[object Object]: Response code 404 (Not Found)`.

`.arguments('<command')` changes parse result of commander option, remove it would work I think.

Reply:
well, but it does work 🤔

Reply:
You are right, I tested with this and a little wrong still exist.

``` js
program
  .arguments('<command> [options]')
  .action((cmd) => {
```